projects/schematics/src/migrations/5_0/constructor-deprecations/data/cart-lib-checkout.migration.ts

```diff
@@ -0,0 +1,2679 @@
+      },
+    ],
+    addParams: [
+      {
+        className: 'CartValidationFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'CmsGuardsService',
+    importPath: '@spartacus/storefront',
+    deprecatedParams: [
+      {
+        className: 'CmsComponentsService',
+        importPath: '@spartacus/storefront',
+      },
+      {
+        className: 'Injector',
+        importPath: '@angular/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'Injector',
+        importPath: '@angular/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'UnifiedInjector',
+        importPath: '@spartacus/core',
+      },
+    ],
+  },
+  {
+    class: 'ConsignmentTrackingComponent',
+    importPath: '@spartacus/storefront',
+    deprecatedParams: [
+      {
+        className: 'UserOrderService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'ModalService',
+        importPath: '@spartacus/storefront',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'UserOrderService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'OrderHistoryFacade',
+        importPath: '@spartacus/order/root',
+      },
+    ],
+  },
+  {
+    class: 'MiniCartComponent',
+    importPath: '@spartacus/storefront',
+    deprecatedParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'MiniCartComponentService',
+        importPath: '@spartacus/cart/base/components/mini-cart',
+      },
+    ],
+  },
+  {
+    class: 'OrderCancellationService',
+    importPath: '@spartacus/storefront',
+    deprecatedParams: [
+      {
+        className: 'OrderDetailsService',
+        importPath: '@spartacus/order/components',
+      },
+      {
+        className: 'UserOrderService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'RoutingService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'GlobalMessageService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'UserOrderService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'OrderHistoryFacade',
+        importPath: '@spartacus/order/root',
+      },
+    ],
+  },
+  {
+    class: 'OrderDetailsService',
+    importPath: '@spartacus/storefront',
+    deprecatedParams: [
+      {
+        className: 'UserOrderService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'RoutingService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'UnifiedInjector',
```

Comment:
did not exist in release/4.0.0 and release/4.1.x when this service was in the storefrontlib